.alexrc.yml

```diff
@@ -0,0 +1,4 @@
+allow:
```

Comment:
These terms are potentially insensitive when used to refer to people, but are helpful in the context of discussing code, e.g.

"invalid data"
"special characters"
"executed a command"